README.md

```diff
@@ -1080,6 +1080,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Yamini Kaushal](https://github.com/Yamini09-code)
   - [Yamini Singh](https://github.com/yamini-singh169)
   - [Yanuka Deneth](https://github.com/yanukadeneth99)
+  - [Yash Gaur](https://github.com/THEYASHGAUR)
```

Comment:
Your name is in the wrong order.
You have to add your name after Yash Choudhary.

```
 - [Yash Choudhary](https://github.com/yashc9602)
 - [Yash Gaur](https://github.com/THEYASHGAUR)
```